daprdocs/content/en/js-sdk-docs/_index.md

```diff
@@ -0,0 +1,193 @@
+  });
+
+  //GET Request
+  const response = await client.invoker.invoke(SERVICE_TO_INVOKE, METHOD_TO_INVOKE, HttpMethod.GET);
+}
+```
+- For a full guide on service invocation visit [How-To: Invoke a service]({{< ref howto-invoke-discover-services.md >}}).
+
+### Save & get application state
+
+```javascript
+import { DaprClient } from "@roadwork/dapr-js-sdk/http";
+
+const daprHost = "127.0.0.1";
+const daprPort = "50050"; 
+
+async function start() {
+  const client = new DaprClient(daprHost, daprPort);
+
+  //Save state
+  const response = await client.state.save(STATE_STORE_NAME, [
+    {
+      key: FIRST_KEY_NAME 
+      value: FIRST_VALUE
+    },
+    {
+      key: SECOND_KEY_NAME,
+      value: SECOND_VALUE
+    },
+    {
+      key: THIRD_KEY_NAME
+      value: THIRD_VALUE
+    }
+  ]);
+
+  //Get State
+  const response = await client.state.get(STATE_STORE_NAME, FIRST_KEY_NAME);
+
+  //Get Bulk State
+  const response = await client.state.getBulk(STATE_STORE_NAME, [ FIRST_KEY_NAME, SECOND_KEY_NAME ]);
+
+  //Delete State
+  const response = await client.state.delete(STATE_STORE_NAME, FIRST_KEY_NAME);
+}
+```
+- For a full list of state operations visit [How-To: Get & save state]({{< ref howto-get-save-state.md >}}).
+
+### Publish & subscribe to messages
+
+##### Publish messages
+
+```javascript
+import { DaprClient } from "@roadwork/dapr-js-sdk/http";
+
+const daprHost = "127.0.0.1";
+const daprPort = "50050"; 
+
+async function start() {
+  const client = new DaprClient(daprHost, daprPort);
+
+  const response = await client.pubsub.publish(PUBSUB_NAME, TOPIC_NAME, { messsage });
+}
+```
+
+##### Subscribe to messages
+
+```javascript
+import { DaprClient } from "@roadwork/dapr-js-sdk/http";
+
+const daprHost = "127.0.0.1";
+const daprPort = "50050"; 
+
+async function start() {
+  const client = new DaprClient(daprHost, daprPort);
+
+  const response = await server.pubsub.subscribe(PUBSUB_NAME, TOPIC_NAME, async (data) => console.log(`Got Data: ${JSON.stringify(data)}`));
+}
+```
+
+- For a full list of state operations visit [How-To: Publish & subscribe]({{< ref howto-publish-subscribe.md >}}).
+
+### Interact with bindings
+
+**Output Bindings**
+```javascript
+import { DaprClient } from "@roadwork/dapr-js-sdk/http";
+
+const daprHost = "127.0.0.1";
+const daprPort = "50050"; 
+
+async function start() {
+  const client = new DaprClient(daprHost, daprPort);
+
+  const response = await client.binding.send(BINDING_NAME, BINDING_OPERATION, { message });
+}
+```
+
+**Input Bindings**
+```javascript
+import { DaprServer } from "@roadwork/dapr-js-sdk/http";
+
+const daprHost = "127.0.0.1";
+const daprPort = "50050"; 
+const daprInternalServerPort = "50051"; // App Port of this Example Server
+
+async function start() {
+  const server = new DaprServer(daprHost, daprPort, daprInternalServerPort);
+
+  const response = await server.binding.receive(BINDING_NAME, async (data) => console.log(`Got Data: ${JSON.stringify(data)}`));
```

Comment:
For this to work, the server should start listening with `await server.startServer()`;

Reply:
Done